src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+#Telegram bot information
+BOT_NAME=HarrietBicherStouBot
+BOT_TOKEN=5216818148:AAEvMW6Bjw6eXMf7YqUCPpa-QIcQswwFQFU
```

Comment:
Kindly, go through [this](https://stackoverflow.com/questions/12473537/naming-convention-for-properties-in-property-file) question on SO and decide if the variables still need to be named in UPPER_UNDERSCORE_CASE

PS: Could you please leave a blank line at the end of each file?